articles/openshift/oshift-how-portworx-snapshots.md

```diff
@@ -0,0 +1,347 @@
+        requests:
+          storage: 2Gi" | oc create -f -
+    ```
+
+### Group snapshot
+
+Portworx has the ability to take a group snapshot of all PVCs that match a specified label. This is useful to ensure backups of multiple PVCs are taken at the same time. To take a group snapshot, use the `GroupVolumeSnapshot` resource. The following example creates three PVCs with the same label and leverages the `GroupVolumeSnapshot` resource to back them up:
+
+```none
+for i in {0..2}
+do
+echo "apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: group-snapshot-test-$i
+  namespace: test-snapshot-1
+  annotations: 
+    volume.beta.kubernetes.io/storage-class: portworx-repl1
+  labels:
+    purpose: "group-test"
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 2Gi" | oc create -f -
+done
+```
+
+1. Create the `GroupVolumeSnapshot` resource targeting the label of your desired PVCs, in our example, the label `purpose=group-test`:
+
+    ```none
+    echo "apiVersion: stork.libopenstorage.org/v1alpha1
+    kind: GroupVolumeSnapshot
+    metadata:
+      name: test-groupsnapshot
+    spec:
+      pvcSelector:
+        matchLabels:
+          purpose: group-test" | oc create -f -
+    ```
+
+2. This will create a `VolumeSnapshot` object for each PVC that it matches. To find the name of the volume snapshots you can describe the `GroupVolumeSnapshot` object:
+
+    ```none
+    oc describe groupvolumesnapshot | grep "Volume Snapshot Name"
+    
+        Volume Snapshot Name:  test-groupsnapshot-group-snapshot-test-2-61d7ca5e-ee87-11e9-8422-fa163e52fd0e
+        Volume Snapshot Name:  test-groupsnapshot-group-snapshot-test-1-61d7ca5e-ee87-11e9-8422-fa163e52fd0e
+        Volume Snapshot Name:  test-groupsnapshot-group-snapshot-test-0-61d7ca5e-ee87-11e9-8422-fa163e52fd0e
+    ```
+
+3. To restore any of the PVCs, create a PVC object and insert the relevant snapshot name:
+
+    ```none
+    echo "apiVersion: v1
+    kind: PersistentVolumeClaim
+    metadata:
+      name: pvc-restore-1
+      namespace: test-snapshot-1
+      annotations:
+        snapshot.alpha.kubernetes.io/snapshot: <snapshot name>
+    spec:
+      accessModes:
+         - ReadWriteOnce
+      storageClassName: stork-snapshot-sc
+      resources:
+        requests:
+          storage: 2Gi" | oc create -f -
+    ```
+
+### Cloud snapshots
+
+Portworx enables you to backup snapshots to cloud storage. We can provide a bucket for the cluster on our Cloud Storage service and set up a cloud credential that you can use. If you'd like to configure snapshots to be sent to your own storage, see <https://docs.portworx.com/reference/cli/credentials/#overview> for how to set this up. 
+
+> [!TIP]
+> To find the ID of the cloud credentials provided with your cluster run:
+> `PX_POD=$(kubectl get pods -l name=portworx -n kube-system -o jsonpath='{.items[0].metadata.name}')`
+> `kubectl exec $PX_POD -n kube-system -- /opt/pwx/bin/pxctl credentials list`
```

Comment:
Can you elaborate briefly on what exactly this is doing in the article, so its not just some magic command we run.